src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEndpoints.cs

```diff
@@ -19,6 +19,177 @@ public enum EnvironmentNames
+        /// </summary>
+        /// <param name="testEndpoint">endPoint that needs to be updated according to connection string</param>
+        /// <param name="connString">User provided connection string</param>
+        internal TestEndpoints(TestEndpoints testEndpoint, ConnectionString connString): this(testEndpoint)
+        {
+            //FIND OUT MAPPING FROM CONNECTION STRING
+            //if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.RdfePortalUriKey)))
+            //{
+            //    ServiceManagementUri = new Uri(connString.GetValue(ConnectionStringKeys.RdfePortalUriKey));
+            //}
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.BaseUriKey)))
+            {
+                ResourceManagementUri = new Uri(connString.GetValue(ConnectionStringKeys.BaseUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.GraphUriKey)))
+            {
+                GraphUri = new Uri(connString.GetValue(ConnectionStringKeys.GraphUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.GalleryUriKey)))
+            {
+                GalleryUri = new Uri(connString.GetValue(ConnectionStringKeys.GalleryUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.AADAuthenticationEndpointKey)))
+            {
+                AADAuthUri = new Uri(connString.GetValue(ConnectionStringKeys.AADAuthenticationEndpointKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.RdfePortalUriKey)))
+            {
+                RdfePortalUri = new Uri(connString.GetValue(ConnectionStringKeys.RdfePortalUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.IbizaPortalUriKey)))
+            {
+                IbizaPortalUri = new Uri(connString.GetValue(ConnectionStringKeys.IbizaPortalUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.DataLakeStoreServiceUriKey)))
+            {
+                DataLakeStoreServiceUri = new Uri(connString.GetValue(ConnectionStringKeys.DataLakeStoreServiceUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.DataLakeAnalyticsJobAndCatalogServiceUriKey)))
+            {
+                DataLakeAnalyticsJobAndCatalogServiceUri = new Uri(connString.GetValue(ConnectionStringKeys.DataLakeAnalyticsJobAndCatalogServiceUriKey));
+            }
+
+            if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.AADTokenAudienceUriKey)))
+            {
+                AADTokenAudienceUri = new Uri(connString.GetValue(ConnectionStringKeys.AADTokenAudienceUriKey));
+            }
+
+            //if (!string.IsNullOrEmpty(connString.GetValue(ConnectionStringKeys.RawGraphTokenKey)))
+            //{
```

Comment:
I need help here to figure out what part of the connection string needs to be mapped.
So if you can tell me what those endPoints should be mapped to, so that I can update it accordingly.

---
In reply to: [78469501](https://github.com/Azure/azure-sdk-for-net/pull/2328#discussion_r78469501) [](ancestors = 78469501)